processor/src/decoder/block_stack.rs

```diff
@@ -137,6 +137,14 @@ impl BlockInfo {
         }
     }
 
+    /// Returns ONE if this block is a SYSCALL block; otherwise returns ZERO.
+    pub fn is_syscall(&self) -> Felt {
```

Comment:
optional nit: this function can be `const`

Reply:
Thanks! I made this, and a couple of similar functions `const`.